### PR TITLE
Hide "Unpin" button in repo header

### PR DESCRIPTION
**Pull request by cheap-glitch**
https://github.com/refined-github/refined-github/issues/5398#issuecomment-1036112206

## Test URLs

Any page of a repo pinned to your profile

## Screenshot

**Before**

before

**After**

after

### CODE REVIEW

**Review by fregante — APPROVED** [review]

**Comment by kidonng** [comment]
Note this hides the button completely, not just when it shows "Unpin".

**Comment by cheap-glitch** [comment]
> Note this hides the button completely, not just when it shows "Unpin".

Can it show something else? If you click it, you get redirected to your profile page anyway.

**Comment by kidonng** [comment]
> Can it show something else?

Let me rephrase a bit 🥲 _it also hides the "Pin" button_

**Comment by cheap-glitch** [comment]
> > Can it show something else?
> 
> Let me rephrase a bit smiling_face_with_tear _it also hides the "Pin" button_

Aaah ok, because my pins are always full I didn't realize there was a complementary "Pin" button :facepalm: 

Dropping some CSS to revert this just in case:

```css
.pagehead-actions.pagehead-actions [data-test-selector="pin-repo-button"] {
	display: inline-block !important;
}
```
